requirements.txt

```diff
@@ -3,3 +3,4 @@
 --trusted-host imos-artifacts.s3-website-ap-southeast-2.amazonaws.com
 -c constraints.txt
 -e .
+fuzzywuzzy
```

Comment:
@lwgordonimos can I do this?

Reply:
Nope, you need to add it to setup.py, in the `INSTALL_REQUIRES` list.

Reply:
done